example/lib/main.dart

```diff
@@ -25,7 +25,7 @@ class _MyAppState extends State<MyApp> {
     String platformVersion;
     // Platform messages may fail, so we use a try/catch PlatformException.
     try {
-      platformVersion = await DocumentDetector.platformVersion;
+      platformVersion = await DocumentDetector.detect('asdf');
```

Comment:
Please check

Reply:
The camera page will open and pass the image path. For now, a temporary string was used for testing.